src/Microsoft.Diagnostics.Runtime/src/Common/ClrHandleKind.cs

```diff
@@ -47,6 +47,8 @@ public enum ClrHandleKind
         /// <summary>
         /// Strong handle used internally for book keeping.
         /// </summary>
-        SizedRef = 8
+        SizedRef = 8,
+
+        WeakWinRT = 9,
```

Comment:
Need a comment on this briefly describing what it is.

Reply:
A WeakWinRT handle is a weak WinRT handle, that's all... How could you expect the users to understand the following:

> WinRT weak reference handles hold two different types of weak handles to any RCW with an underlying COM object that implements IWeakReferenceSource.  The object reference itself is a short weak handle to the RCW...

?

Reply:
Just having `/// Weak WinRT handle.` is enough here.  I was only objecting to not having a comment on the enum at all because it can cause a build warning with some settings.  We don't have to fully explain what it is here.